templates/config.cli/12-syslog.epp

```diff
@@ -0,0 +1,23 @@
+<%- |
+String $prefix,
+Boolean $syslog,
+String $syslog_app_name,
+String $syslog_facility,
+String $syslog_hostname,
+String $syslog_level,
+String $syslog_port,
+Stdlib::Host $syslog_server_address,
+String $syslog_format
+| -%>
+<%- if $syslog { -%>
+if (outcome != success) of <%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG:read-resource
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG:add
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=app-name, value=<%= $syslog_app_name %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=facility, value=<%= $syslog_facility %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=hostname, value=<%= $syslog_hostname %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=level, value=<%= $syslog_level %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=port, value=<%= $syslog_port %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=server-address, value=<%= $syslog_server_address %>)
+<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG/:write-attribute(name=syslog-format, value=<%= $syslog_format %>)
+end-if
```

Comment:
Should make it so that if `$syslog` is `false` the syslog handler is removed.  Maybe this:

```
<%- } else { -%>
if (outcome == success) of <%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG:read-resource
<%= $prefix -%>/subsystem=logging/syslog-handler=SYSLOG:remove
end-if
```